tests/pom.xml

```diff
@@ -43,6 +43,8 @@ Test cases are run against Docker images of Hono server + (Apache Qpid Dispatch
     <!-- should be set to false if testing against a registry that doesn't support GW mode -->
     <deviceregistry.supportsGatewayMode>true</deviceregistry.supportsGatewayMode>
     <deviceregistry.credentials.supportsClientContext>true</deviceregistry.credentials.supportsClientContext>
+    <!-- should be set to true if testing against a registry that supports search devices operation -->
+    <hono.deviceregistry.supportsSearchDevices>false</hono.deviceregistry.supportsSearchDevices>
```

Comment:
maybe we can keep the name of this feature flag in line with the other two flags that we already use?